src/app/public/modules/repeater/repeater.component.ts

```diff
@@ -21,17 +26,37 @@ import {
   SkyRepeaterService
 } from './repeater.service';
 
+import {
+  SkyRepeaterAdapterService
+} from './repeater-adapter.service';
+
 @Component({
   selector: 'sky-repeater',
   styleUrls: ['./repeater.component.scss'],
   templateUrl: './repeater.component.html',
-  providers: [SkyRepeaterService]
+  providers: [SkyRepeaterService, SkyRepeaterAdapterService],
+  viewProviders: [DragulaService]
 })
 export class SkyRepeaterComponent implements AfterContentInit, OnChanges, OnDestroy {
 
   @Input()
   public activeIndex: number;
 
+  @Input()
+  public set draggable(isDraggable: boolean) {
```

Comment:
I'm not sure `draggable` is the right name for this input. For example, a user doesn't have to "drag" anything to use this feature. I wonder if it should be something more like `enableItemReordering`, `itemReorder` ?

Reply:
Done. Changed to `reorderable`